core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -805,12 +809,12 @@ class TimeSeriesShard(val ref: DatasetRef,
 
   private def updateGauges(): Unit = {
     assertThreadName(IngestSchedName)
-    shardStats.bufferPoolSize.set(bufferPools.valuesArray.map(_.poolSize).sum)
-    shardStats.indexEntries.set(partKeyIndex.indexNumEntries)
-    shardStats.indexBytes.set(partKeyIndex.indexRamBytes)
-    shardStats.numPartitions.set(numActivePartitions)
+    shardStats.bufferPoolSize.update(bufferPools.valuesArray.map(_.poolSize).sum)
```

Comment:
This comment is misplaced, I couldnt add a line there since the lines didnt change.

We need a marker which calls out number of partitions touched by the query.
This can be added in lookupPartitions.